packages/core/src/App/Components/Layout/Header/account-actions.jsx

```diff
@@ -121,7 +121,7 @@ const AccountActions = React.memo(
                             <Button
                                 className='acc-info__button'
                                 has_effect
-                                text={localize('Deposit')}
+                                text={is_cra ? localize('withdrawal') : localize('Deposit')}
```

Comment:
```suggestion
                                text={is_cra ? localize('Withdrawal') : localize('Deposit')}
```

Reply:
i think it's a capital `W` in the design.😄

Reply:
you are right, I'll change that :)))